### PR TITLE
refactor: Remove Pools

### PR DESCRIPTION
**Pull request by FranRom**
This PR removes all the code Pool related.

### CODE REVIEW

File: src/config/abi/masterchef.json [hunk 1]
```diff
@@ -293,13 +293,6 @@
     "stateMutability": "nonpayable",
     "type": "function"
   },
```
Comment (FranRom) [inline src/config/abi/masterchef.json:295]: Do we need this abi yet?

File: src/config/abi/sousChef.json [hunk 2]
```diff
@@ -96,25 +96,6 @@
     "type": "function",
     "constant": true
   },
```
Comment (FranRom) [inline src/config/abi/sousChef.json:98]: Same here, do we need it? or can I delete it safely...

File: src/config/abi/sousChefBnb.json [hunk 3]
```diff
@@ -95,7 +95,6 @@
     "stateMutability": "view",
     "type": "function"
   },
```
Comment (FranRom) [inline src/config/abi/sousChefBnb.json:97]: Same here, keep it or delete it?

File: src/config/abi/masterchef.json (hunk 1 above)
Comment (jffernandez) [reply src/config/abi/masterchef.json:295]: No, this can be deleted.

File: src/config/abi/sousChef.json (hunk 2 above)
Comment (jffernandez) [reply src/config/abi/sousChef.json:98]: Same, we only call our contracts.

File: src/config/abi/sousChefBnb.json (hunk 3 above)
Comment (jffernandez) [reply src/config/abi/sousChefBnb.json:97]: Delete.

**Review by jffernandez — APPROVED** [review]

**Comment by FranRom** [comment]
I'm going to delete more in next PRs, too many files already..

**Pull request merged**